Lib/asyncio/queues.py

```diff
@@ -200,7 +202,7 @@ async def get(self):
                     # the call.  Wake up the next in line.
                     self._wakeup_next(self._getters)
                 raise
-            if self._shutdown_state == _QueueState.shutdown_immediate:
+            if self._shutdown_state is not _QueueState.alive:
```

Comment:
@YvesDup what is the reason for this? `_shutdown_state is not _QueueState.alive` will be tested in the next iteration of the `while` loop: we only need to test if the queue is in the shutdown-immediate state

Reply:
It is a mistake. Ok